openupgrade_scripts/scripts/product/18.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,100 @@
+---Models in module 'product'---
+new model product.combo
+new model product.combo.item
+
+# NOTHING TO DO: moved in from pos
```

Comment:
why not rename `pos.combo` to `product.combo` and `pos.combo.line` to `product.combo.item`?

Reply:
I didn't want to dive into this yet as we don't need it for pure product migration. Left a note that this should be revisited when when doing the pos migration